test/e2e/next-test-lib/next-modes/next-deploy.ts

```diff
@@ -89,6 +102,37 @@ export class NextDeployInstance extends NextInstance {
     // no-op as the deployment is created during setup()
   }
 
+  public async destroy(): Promise<void> {
+    if (this.isDestroyed) {
+      throw new Error(`next instance already destroyed`)
```

Comment:
Consider a more verbose error.

```suggestion
      throw new Error(`Next.js deploy instance already destroyed`)
```